src/ducks/Watchlists/Widgets/TopBar/TopBar.module.scss

```diff
@@ -47,3 +47,67 @@
   align-items: center;
   justify-content: space-between;
 }
+
+.commentsWrapper {
+  background-color: var(--white);
+  border-top: 1px solid var(--porcelain);
+  border-top-left-radius: 4px;
+  border-left: 1px solid var(--porcelain);
+  border-bottom-left-radius: 4px;
+  width: 363px;
+  z-index: 11;
+  position: fixed;
+  top: var(--screener-top-panel-height);
+  bottom: 0;
+  right: 0;
+  transition: transform 0.3s ease-out;
+  transform: scaleX(0);
+  transform-origin: top right;
+  user-select: none;
+  padding: 16px 24px;
+}
+
+.active {
+  transform: scaleX(1);
+  animation: growPartially 0.35s ease-out;
+}
+
+.background {
+  position: fixed;
+  z-index: 9;
+  background-color: $rhino;
+  opacity: 0.6;
+  top: var(--screener-top-panel-height);
+  left: 0;
+  right: 0;
+  bottom: 0;
+  animation: appearPartially 0.4s ease-in-out;
+}
+
+.closeWrapper {
+  z-index: 10;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
```

Comment:
Use [san-webkit](https://github.com/santiment/san-webkit)'s classes: `className="btn row v-center border"`. Also, take a look at the storyook's [`Button`](https://github.com/santiment/san-webkit/blob/master/stories/Button/index.stories.svelte) page and style rules